src/pyskindose/constants.py

```diff
@@ -146,12 +147,16 @@
 PLOT_ASPECTMODE_PLOT_DOSEMAP = 'data'
 
 PLOT_ASPECTMODE_PLOT_PROCEDURE = 'cube'
+PLOT_ORDER_STATIC = ['right', 'back', 'left', 'front']
 
 PLOT_PROCEDURE_AXIS_RANGE_X = [-300, 300]
 PLOT_PROCEDURE_AXIS_RANGE_Y = [-300, 300]
 PLOT_PROCEDURE_AXIS_RANGE_Z = [-300, 300]
-
+PLOT_GROUND_SHIFT_X_STATIC = 50
```

Comment:
Perhaps we could add comments specifying where these "magic" numbers come from? Might be helpful for later

Reply:
Resolved in 41fa517539332949b571d0a07ece664a7ffbc0b7